### ISSUE TITLE
Additional decoration

### ISSUE THREAD

**Issue by masics** [t9md/atom-quick-highlight#10]
For some reason in my configuration I barely see the selection. I tried to use "highlight and box" but it looks exactly the same (most likely as "box").
Could you please check it out?

**Comment by gouegd** [comment]
Hi ! I was wondering something similar and dug up a bit.
The highlight or box styling only applies to highlights that are toggled manually, not to the automatic (optional) word highlight, that's why you don't see a difference between the two.

Here's how I modified the automatic word highlighting to suit my config (box-selection is the style applied for automatic highlightings):
```less
atom-text-editor::shadow {
  .quick-highlight.box-selection .region {
    border-radius: 0;
    border-color: red;
  }
}
```

**Comment by t9md** [comment]
Sorry, I didn't understand this issue till now.
Since I'm not using selection highlight.

I confirmed selection highlight is VERY dim.
This become issue from when depending less variable `@syntax-result-marker-color` was changed(used this color for border of selection).

Will release update soon.
And will add instruction for "how to modify style" in README.md

For the selection color style, current box-selection is NOT very intuitive so will change it to bare `selection`
